crypto/fipsmodule/bn/rsaz_exp_x2.c

```diff
@@ -0,0 +1,623 @@
+/*
+ * Copyright 2020-2021 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2020-2021, Intel Corporation. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ *
+ * Originally written by Sergey Kirillov and Andrey Matyukov.
+ * Special thanks to Ilya Albrekht for his valuable hints.
+ * Intel Corporation
+ *
+ */
+#ifndef RSAZ_512_ENABLED
+/*
+ * This appears in openssl as just:
+ *
+ *   NON_EMPTY_TRANSLATION_UNIT
+ *
+ * This macro is defined in include/openssl/macros.h like so:
+ *
+ *   // Sometimes OPENSSL_NO_xxx ends up with an empty file and some
+ *   // compilers don't like that.  This will hopefully silence them.
+ *   # define NON_EMPTY_TRANSLATION_UNIT static void *dummy = &dummy;
+ */
+static void *__dummy = &__dummy;
+#else
```

Comment:
looking at `rsaz_exp.c` it seems like we don't need this hack, can we please get rid of it?

Reply:
I added this because of Apple build failures early on. I'm happy to remove it to see if it is still needed.